samples/sampler/stories/default/QuickGuidePanels.js

```diff
@@ -13,31 +20,72 @@ export default {
 	component: 'QuickGuidePanels'
 };
 
-export const _QuickGuidePanels = (args) => (
-	<QuickGuidePanels
-		current={args['current']}
-		nextButtonVisibility={args['nextButtonVisibility']}
-		onBack={action('onBack')}
-		onChange={action('onChange')}
-		onClose={action('onClose')}
-		onNextClick={action('onNextClick')}
-		onPrevClick={action('onPrevClick')}
-		prevButtonVisibility={args['prevButtonVisibility']}
-		total={args['total']}
-	>
-		<QuickGuidePanels.Panel>
-			This is Panel 0
-		</QuickGuidePanels.Panel>
-		<QuickGuidePanels.Panel>
-			<div style={{height: '100%', width: '100%', backgroundColor: 'grey'}}>
-				This is Panel 1
-			</div>
-		</QuickGuidePanels.Panel>
-		<QuickGuidePanels.Panel aria-label={'aria test'}>
-			This is Panel 2
-		</QuickGuidePanels.Panel>
-	</QuickGuidePanels>
-);
+export const _QuickGuidePanels = (args) => {
+	const screenWidth = window.screen.width;
+	const screenHeight = window.screen.height;
+	const widthratio = screenWidth / 1920;
+	const heightratio = screenHeight / 1080;
+
+	return (
+		<>
+			<Panel style={{margin: ri.scaleToRem(240)}}>
+				<Button size="small" style={{marginLeft: ri.scaleToRem(64), marginBottom: ri.scaleToRem(160)}}>Button</Button>
```

Comment:
Since we have css, why don't we use classname instead of inline style?

Reply:
I used classname instead of inline style. However, to apply margin values, I need to set focusEffect to "static". If focusEffect="expand", the margin value cannot be overridden because focusExpand is not in the publicClassNames list.